app/assess/templates/sub_criteria.html

```diff
@@ -0,0 +1,105 @@
+
+    <div class="govuk-grid-row s26-headline-container">
+        <div class="s26-headline-info">
+            <div>
+                <h1 class="govuk-heading-l">Fund: <span>{{ fund['name'] }}</span></h1>
+                <h1 class="govuk-heading-l">Project reference: <span>LKMBNS</span></h1>
+            </div>
+
+            <p class="govuk-body-l">Project name: <span>Save the White Swan, Dursley</span></p>
+            <p class="govuk-body-l">Total funding requested: <span>£98,400</span></p>
+            <p class="govuk-body-l">Submitted: <span>11/06/22</span></p>
+        </div>
+    </div>
+
+    <div class="govuk-grid-row">
+
+        <div class="s26-content-container">
+
+
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-one-third">
+                    <a href="#" class="govuk-back-link">Back to <span class="govuk-!-font-weight-bold">assessment
+                            overview</span></a>
+                    <h1 class="govuk-heading-l scoring-heading">Score engagement</h1>
+
+                </div>
+            </div>
+
+            <div class="govuk-grid-column-full">
+
+                <div class="govuk-grid-column-one-third">
+
+                    <nav class="assessment-navigation">
+                        <ul class="govuk-list">
+                            <li><a class="govuk-link--no-visited-state"
+                                    href="/s26-assessment/commenting/community-engagement">Community
+                                    engagement</a></li>
+                            <li><a class="govuk-link--no-visited-state"
+                                    href="/s26-assessment/commenting/local-support">Local support</a></li>
+                        </ul>
+
+                        <ul class="govuk-list">
+                            <li>Score the subcriteria</li>
+                        </ul>
+                    </nav>
+                </div>
+<!--  -->
+{{scores_justification(scores_submitted, form, latest_score, application_id, sub_criteria_id, COF_score_list)}}
+<!--  -->
+                    <h2 class="govuk-heading govuk-heading-l" id="comments-heading">Comments</h2>
+                    <section class="comment-section">
+                        <h3 class="govuk-heading-m">Community engagement</h3>
+
+                        <p class="govuk-body-s">There are currently no comments&nbsp;for this section.
+                            <!--<a href="#" class="govuk-link govuk-link--no-visited-state">Add a comment</a>-->
+                        </p>
+
+                    </section>
+                    <section class="comment-section">
+                        <h3 class="govuk-heading-m">Local support</h3>
+
+                        <p class="govuk-body-s">There are currently no comments&nbsp;for this section.
+                            <!--<a href="#" class="govuk-link govuk-link--no-visited-state">Add a comment</a>-->
```

Comment:
We can use Jinja comments instead of HTML comment syntax? https://jinja.palletsprojects.com/en/3.1.x/templates/#comments